PlaywrigthUITests/Tests/WebTableTests.cs

```diff
@@ -32,15 +33,121 @@ public async Task VerifyTableVisible2()
             await DemoQAWebTablesPage.VerifyTableRowContent("Last Name", "Cantrell");
 
             await Page.GetByRole(AriaRole.Button, new() { Name = "Add" }).ClickAsync();
-            await DemoQAWebTablesPage.VerifyPopupVisible();
+            await DemoQAWebTablesPage.VerifyPopupVisible(true);
             await DemoQAWebTablesPage.VerifyFirstNameVisible();
         }
 
         //TODO: automate test cases
         //Check any mandatory field
+        [Test]
+        [Description("Verify mandatory fields")]
+        public async Task VerifyDepartmentIsMandatoryField()
+        {
+            //Arrange
+            await DemoQAWebTablesPage.GoToDemoQaWebTablesPage();
+            await DemoQAWebTablesPage.OpenAddPopup();
+            await DemoQAWebTablesPage.VerifyPopupVisible(true);
+            //Act
+            await DemoQAWebTablesPage.FillFirstName("John");
+            await DemoQAWebTablesPage.FillLastName("Doe");
+            await DemoQAWebTablesPage.FillEmail("jon@doe.com");
+            await DemoQAWebTablesPage.FillAge("33");
+            await DemoQAWebTablesPage.FillSalary("555");
+            await DemoQAWebTablesPage.FillDepartment("");
+            await DemoQAWebTablesPage.ClickSubmit();
+            //Assert
+            Assert.That(await DemoQAWebTablesPage.VerifyPopupVisible(false), "The 'Add' popup is still visible.");
```

Comment:
here you verify that popup is "not visible" (which is not true) and on the next line you verify colors of input fields on this popup

Reply:
replaced with RegistrationPopup.WaitForAsync